.gitignore

```diff
@@ -7,6 +7,8 @@
 *.swo
 *#
 
+generated_glm_reports
```

Comment:
to keep with some kind of order, can you move this down to be with the nilearn related stuff? maybe under a `# test` section